publisher-sdk-tests/src/androidTest/java/com/criteo/publisher/network/PubSdkApiIntegrationTest.java

```diff
@@ -385,7 +386,8 @@ private RemoteConfigResponse defaultRemoteConfigResponse() {
         "<html><body style='text-align:center; margin:0px; padding:0px; horizontal-align:center;'><script src=\"%%displayUrl%%\"></script></body></html>",
         "%%adTagData%%",
         "<html><body style='text-align:center; margin:0px; padding:0px; horizontal-align:center;'><script>%%adTagData%%</script></body></html>",
-        true
+        true,
+        null // FIXME: adjust once the remote config is added
```

Comment:
If you want , you can already adjust this by changing the handler CdbMock. This will then make the network tests fail until remote config is added.